### DISCUSSION TITLE
AssertionError: Invalid REFERENCE_WORLD_SIZE in config!

### DISCUSSION THREAD

**Discussion by zkailinzhang** [jinfagang/yolov7#125] [General]
File "//python3.7/site-packages/detectron2/engine/defaults.py", line 674, in auto_scale_workers
    ), "Invalid REFERENCE_WORLD_SIZE in config!"
AssertionError: Invalid REFERENCE_WORLD_SIZE in config!

**Comment by zkailinzhang** [comment]
git clone --branch master

**Comment by VYRION-Ai** [comment]
> 
@zkailinzhang 
what do you mean by   `git clone --branch master`

**Comment by lucasjinreal** [comment]
Set `--num_gpus` following physics devices you have.

**Comment by Usama3059** [comment]
AssertionError: Invalid REFERENCE_WORLD_SIZE in config!   getting the same error, I have set the --num_gpus 0 also tried with --num_gpus 1 on colab still same error

**Comment by VYRION-Ai** [comment]
@Usama3059 in config file rise batch size value

**Comment by Usama3059** [comment]
It's working Thanks @totoadel

**Comment by khakimin** [comment]
> It's working Thanks @totoadel

May I know where do I change the batch size value

**Comment by VYRION-Ai** [comment]
@khakimin what config file you are using?

**Comment by gkpodder** [comment]
@totoadel I'm getting the same error. I'm trying to image segmentation on colab using the yolov7
  File "/content/yolov7/detectron2/detectron2/engine/defaults.py", line 683, in auto_scale_workers
    ), "Invalid REFERENCE_WORLD_SIZE in config!"
AssertionError: Invalid REFERENCE_WORLD_SIZE in config

Thanks in advance!

**Comment by VYRION-Ai** [comment]
@gkpodder what is the config file you are using
